bluemira/base/look_and_feel.py

```diff
@@ -257,34 +262,84 @@ def colourise(string: str, width: int = 73, color: str = "blue") -> str:
 def bluemira_critical(string: str):
     """
     Standard template for bluemira critical errors.
+
+    Parameters
+    ----------
+    string:
+        The string to log at critical level
+
+    Returns
+    -------
+    :
+        The critical error logged string
     """
     return LOGGER.critical(colourise(f"CRITICAL: {string}", color="darkred"))
 
 
 def bluemira_error(string: str):
     """
     Standard template for bluemira errors.
+
+    Parameters
+    ----------
+    string:
+        The string to log at error level
+
+    Returns
+    -------
+    :
+        The error logged string
     """
     return LOGGER.error(colourise(f"ERROR: {string}", color="red"))
 
 
 def bluemira_warn(string: str):
     """
     Standard template for bluemira warnings.
+
+    Parameters
+    ----------
+    string:
+        The string to log at warning level
+
+    Returns
+    -------
+    :
+        The warning logged string
```

Comment:
I actually think these return None so the return itself should probably be removed

Reply:
Right yeah